packages/crayons-core/src/components/icon/readme.md

```diff
@@ -1,18 +1,24 @@
 # Icon (fw-icon)
-fw-icon displays an icon-sized image that imparts meaning to the component it is associated with.
 
-## Demo
+"fw-icon/FwIcon" is a renderer of a svg file that displays an icon-sized image which imparts meaning to the html component it is associated with.
+Following features are available as part of the implementation of fw-icon..
+
+1. Enable fw-icon as an optimised renderer for SVG with built-in functions such as Intersection-Observer and Fetch-API Memoization.
+   Go through the docs to understand the various props it supports.
+2. React-Support as part of Caryons Release. 
+
+## Exampes Live
 
 ```html live
-<fw-icon name="add-contact" size="18" color="green"></fw-icon>
+<fw-icon name="Header" size="38" color="blue" nolazy></fw-icon>
```

Comment:
This would be a good opportunity to rename the icon `Header` -> `header`

Reply:
Done. Closed.